### PR TITLE
MINOR : fix sidebar icon sizes and remove svg utils file

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes:

- fix sidebar icon sizes
- fix navba and help dropdown icon sizes with spacing.
- remove `svg.utils` file and used alternative `ReactComponent` icon where it's been used.

image

image

image

#
### Type of change:

- [x] Bug fix
- [x] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/assets/svg/globalsearch.svg
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/assets/svg/globalsearch.svg:1]: can we rename with snake-case?

File: openmetadata-ui/src/main/resources/ui/src/assets/svg/lampcharge.svg
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/assets/svg/lampcharge.svg:1]: same snake-case

File: openmetadata-ui/src/main/resources/ui/src/assets/svg/whatsNew.svg
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/assets/svg/whatsNew.svg:1]: snake-case

**Review by chirag-madlani — DISMISSED** [review]

**Comment by Ashish8689** [comment]
@chirag-madlani  changes made as per comments!

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**